### ISSUE TITLE
Support of testcaferc.js is from version 1.6 - missing in docs

### ISSUE THREAD

**Issue by yoavws** [DevExpress/testcafe#6915]
### What is your Scenario?

The documentation of global hooks lead me to non supported .js config file.
After a lot of searches I found this reference: https://giters.com/DevExpress/testcafe/issues/6855
which indicates that the support for testcaferc.js config file is from version 1.6

please update the documentation with the minimum supporting version.

In addition the global hook example at:https://testcafe.io/documentation/402638/reference/configuration-file#hooks
is not correct (missing "}")

module.exports = {
  hooks: {
    fixture: {
      before: async ctx => {
        utils.populateDb(ctx.dbName);
      },
      after: async ctx => {
        utils.dropDb(ctx.dbName);
    },    
**},** //missing
    test: {
      before: async t => {
      **//  .useRole(admin;**  //will not be recognised and missing ")"
      },
      after: async t => {
        await t.click('#delete-data');
    },
  },
**}** //missing
};

### What is the Current behavior?

documentations is missing "supporting since ver x"

### What is the Expected behavior?

Documentation should be updated

### What is your public website URL? (or attach your complete example)

I found out the issue from this question: https://giters.com/DevExpress/testcafe/issues/6855
no public URL as main issue is documentation

### What is your TestCafe test code?

loading  --config-file=./automation/.testcaferc.js with testcafe 1.14 is leading to a loading error on wrong format of config file

### Steps to Reproduce

1.Install testcafe 1.14
2.Execute a testcafe script with  --config-file=./.testcaferc.js 
3.property file loading will fail

### TestCafe version

1.14 

### Command-line arguments

 --config-file=./automation/.testcaferc.js

### Other

After upgrading local testcafe, all worked as expected

**Comment by VasilyStrelyaev** [comment]
Thank you for pointing this out to us. However, we already have a similar issue: https://github.com/DevExpress/testcafe/issues/3675.  
Please subscribe to this issue to keep track of our progress.

I have created a separate issue regarding the example you mentioned: https://github.com/DevExpress/testcafe/issues/6916.
